pages/profiles/snigdharao.md

```diff
@@ -0,0 +1,27 @@
+ # **Snigdha Rao Madhavaram** 
+<p>
+ <img height="250" width="250" src="https://s3.amazonaws.com/mylifecylebuckhyd12/IMG_2569+(1).jpg">
+</p>
+
+Time zone : EDT, -0400
+OS : Mac osX
+BuzzWords :Python, Java, C/C++, JS, Bash, MIPS, Matlab, Linux, Postgres, MySQL, AWS
+
+As a recent graduate with a Master’s in Computer Science from California State University, Long Beach,I am passionate about coding and love to learn new tech stacks .I have  over 2 years of work experience and i love to travel
+
+### Education
+* Master’s in Computer Science, California State University, Long Beach (GPA: 3.7) 
+*  Bachelor’s in Computer Science, Jawaharlal Nehru Technological University (GPA: 4.0)
```

Comment:
Would it be better to remove the extra space before `Bachelor’s in` to make it consistent with the previous line? This does not affect what it looks like in MDwiki, but to makes the raw document neat...